content/departments/engineering/design/templates/brand-XL-template.md

```diff
@@ -0,0 +1,90 @@
+# Brand Design Template: Extra Large
+
+- Estimate: More than a month
+- Notes:
+
+  - When creating the issue, use the label `design-estimated-XL`
+  - All phases for this project should involve a higher level of fidelity. Examples include additional test participants, multiple versions of designs, and more revision, and refinement than designs for smaller projects.
+  - Ideally, design should be given sufficient time in the problem space to explore and research.
+  - Design should be happening at least one iteration before engineering is planned. If user testing is required before or after design, this should be factored into the design time.
+  - These efforts require close collaboration with the Product Manager to scope and define the user problems being solved.
+
+✂️ -------------------------
+
+```markdown:
+## Design process
+
+  - Estimate: More than a month
+  - Weighting factors:
+    - Note: a project of this size requires resources that increase its risk profile.
+    - [HIGH|MEDIUM] risk
+    - [HIGH|MEDIUM|LOW] clarity
+
+  ### Discover
+  Extra large projects should entail more comprehensive research efforts.
+  -[ ] Project brief doc
+    -[ ] Goals
+    -[ ] Who, What, Why
+    -[ ] Constraints
+    -[ ] Constraints (what we can't do)
+    -[ ]	Deliverables (site nav, mood board, content strategy)
+    -[ ]Timeline
+    -[ ]Distribution channels
+    -[ ]Creative direction
+    -[ ]Resources
+  -[ ] Relevant feedback (internal/external)
+  -[ ] Site analytics
+  -[ ] Analogous/competitive research - Link
+
+  ### Define
+  -[ ] Findings, recommendations and direction
+  -[ ] GitHub issue
+
+  ### Design
+  -[ ] Site architecture/User journey
+    -[ ] Review with stakeholders
+  -[ ] Wireframes
+	  -[ ] Review with stakeholders
+    -[ ] Designer review
+  -[ ] Mockup (desktop and mobile)
+    -[ ] Review with stakeholders
+    -[ ] Designer review
+  -[ ] Prototype - Link
```

Comment:
A good way to mark optional items:

```suggestion
  -[ ] Prototype (optional) - Link
```

Reply:
I think for an XL, we would definitely want a prototype, right?